packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/create-projects-area/NewProjectCard.tsx

```diff
@@ -35,6 +38,13 @@ const useNewProjectCardStyles = makeStyles()((theme) => ({
 
 export const NewProjectCard = () => {
   const { classes } = useNewProjectCardStyles();
+
+  const { canCreate } = useContext<ProjectsCapabilitiesContextValue>(ProjectsCapabilitiesContext);
+
+  if (!canCreate) {
+    return null;
+  }
```

Comment:
If the card is displayed, we can use it, no need to check this multiple times

Reply:
Done